pom.xml

```diff
@@ -47,14 +47,14 @@ under the License.
     </modules>
 
     <properties>
-        <flink.version>1.17.0</flink.version>
+        <flink.version>1.16.2</flink.version>
 
         <jackson-bom.version>2.13.4.20221013</jackson-bom.version>
         <junit4.version>4.13.2</junit4.version>
-        <junit5.version>5.9.1</junit5.version>
-        <assertj.version>3.23.1</assertj.version>
-        <testcontainers.version>1.18.2</testcontainers.version>
-        <mockito.version>2.21.0</mockito.version>
+        <junit5.version>5.10.0</junit5.version>
+        <assertj.version>3.24.2</assertj.version>
+        <testcontainers.version>1.19.1</testcontainers.version>
+        <mockito.version>3.4.6</mockito.version>
```

Comment:
out of curiocity: can't we update to the latest 3.x version like 3.12.4?

Reply:
bumped